react/tic-tac-toe/src/utils/setApiHeaders.js

```diff
@@ -0,0 +1,7 @@
+import api from '~config/api';
```

Comment:
you can rename this file to `api.js` because it will contain all the utils refering to the api, what do you think?